std/algorithm/sorting.d

```diff
@@ -774,8 +774,54 @@ private template validPredicates(E, less...)
             && validPredicates!(E, less[1 .. $]);
 }
 
+struct MultiPred( preds... )
```

Comment:
Needs a bit of documentation, you can use the CyberShadow/DAutoTest to preview the documentation http://dtest.thecybershadow.net/artifact/website-81a1cfedcdf480dafc06ec60deab65503fff4213-694e70e92d5b3ea77a96293695cc55dc/web/phobos-prerelease/std_algorithm_sorting.html.